<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OOSubeAramaFormu</class>
 <widget class="QDialog" name="OOSubeAramaFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="twkurulustarihinegoresubearama">
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Cambria&quot;;
}

QDateEdit {
 qproperty-displayFormat: 'dd.MM.yyyy';
 qproperty-calendarPopup: true;
}

</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabIsimeGoreArama">
      <attribute name="title">
       <string>İsime Göre Arama</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblArananDeger">
             <property name="text">
              <string>Aranan Değer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leArananSubeAdi"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grbIsimeGoreAramaSecenekleri">
         <property name="title">
          <string>Arama Şekli</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbIleBaslayan">
            <property name="text">
             <string>İle Baslayan</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbIleBiten">
            <property name="text">
             <string>İle Biten</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbIceren">
            <property name="text">
             <string>İçeren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbYazarkenAra">
            <property name="text">
             <string>Yazarken Ara</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAra">
           <property name="text">
            <string>Ara</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Kuruluş Tarihine Göre Arama</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="lblbaslangictarihi">
             <property name="text">
              <string>Başlangıç Tarihi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="debaslangictarihi"/>
           </item>
           <item>
            <widget class="QLabel" name="lblbitistarihi">
             <property name="text">
              <string>Bitiş Tarihi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="debitistarihi"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="grbAramasecenekleri">
         <property name="title">
          <string>Arama Seçenekleri</string>
         </property>
         <widget class="QRadioButton" name="rbsadecebaslangictarihi">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>sadece başlangıç tarihi</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="rbbitistaihi">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>121</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>sadece bitiş tarihi</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="rbtariharaligi">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Tarih Aralığı</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="btnAramayap">
           <property name="text">
            <string>Arama Yap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btntumsubeler">
           <property name="text">
            <string>Tüm Şubeler</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Şube Kodu</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Şube Adı</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kuruluş Tarihi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Düzelt</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sil</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnKapat">
       <property name="text">
        <string>Kapat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnKapat</sender>
   <signal>clicked()</signal>
   <receiver>OOSubeAramaFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
